mob/src/main/java/com/github/phantazmnetwork/mob/spawner/BasicMobSpawner.java

```diff
@@ -83,16 +188,19 @@ private void setHealth(@NotNull NeuralEntity entity, @NotNull MobModel model) {
     }
 
 
-    private void addGoals(@NotNull PhantazmMob mob) {
-        for (Collection<Goal> group : mob.model().getGoalGroups()) {
-            Collection<NeuralGoal> neuralGroup = new ArrayList<>(group.size());
-            for (Goal goal : group) {
-                neuralGroup.add(goal.createGoal(mob));
-            }
+    private void addGoals(@NotNull ElementContext context, @NotNull DependencyProvider dependencyProvider,
+            @NotNull NeuralEntity entity, @NotNull MobModel model) {
+        Collection<GoalGroup> goalGroups =
+                true ? Collections.emptyList() : context.provide("goalGroups", dependencyProvider);
 
-            GoalGroup goalGroup = new GoalGroup(neuralGroup);
-            mob.entity().addGoalGroup(goalGroup);
+        for (GoalGroup group : goalGroups) {
+            entity.addGoalGroup(group);
         }
     }
 
+    private @NotNull Map<Key, Collection<Skill>> createTriggers(@NotNull ElementContext context,
+            @NotNull DependencyProvider dependencyProvider, @NotNull NeuralEntity entity, @NotNull MobModel model) {
+        return true ? Collections.emptyMap() : context.provide("triggers", dependencyProvider);
```

Comment:
same thing here (actually now that i think about it this is probably waiting for element's update so nevermind)

Reply:
yep